src/components/FileUploader/FileUploader.js

```diff
@@ -12,12 +12,10 @@ export class FileUploaderButton extends Component {
     className: PropTypes.string,
     disableLabelChanges: PropTypes.bool,
     id: PropTypes.string,
-    labelText: PropTypes.string,
-    listFiles: PropTypes.bool,
+    labeltext: PropTypes.string,
```

Comment:
The prop list/names changes look breaking change. To make it clear, we cannot make any breaking changes.

Reply:
The name change was just for removing console errors. I can keep this change exclusive to V2 if that's preferable